### PR TITLE
Fix T4 code generator include template

### PR DESCRIPTION
**Pull request by tchuan**
Add missing references.
Fix compilation error.

### CODE REVIEW

**Comment by biaol-odata** [comment]
@tchuan Can you please specify the detail compilation error you are running into? I can build it clean on my machine. Please note that it requires VS2017 installation for ODataConnectedService.

**Comment by tchuan** [comment]
@biaol-odata I installed the latest version on my VS2017. Here is how to reproduce.

1. Use http://services.odata.org/V4/(S(zvihdypl2mjvnndu2ge4ij4f))/TripPinServiceRW/$metadata as a V4 Endpoint.
2. Select 'Whether to include the T4 files into this project'

`Error		Compiling transformation: The type 'Func<,>' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.	ConsoleApp1	Reference.tt	586	
Error		Compiling transformation: The type 'XmlReader' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Xml.ReaderWriter, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.	ConsoleApp1	Reference.tt	589	
Error		Compiling transformation: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.	ConsoleApp1	Reference.tt	589	
Error		Compiling transformation: Operator '!=' cannot be applied to operands of type 'method group' and '<null>'	ConsoleApp1	Reference.tt	699	
Error		Compiling transformation: foreach statement cannot operate on variables of type 'IEnumerable<IEdmModel>' because 'IEnumerable<IEdmModel>' does not contain a public definition for 'GetEnumerator'	ConsoleApp1	Reference.tt	993	
Error		Compiling transformation: The type 'Enum' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.	ConsoleApp1	Reference.tt	1169	
Error		Compiling transformation: The type 'IEnumerable<>' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.	ConsoleApp1	Reference.tt	1220	
Error		Compiling transformation: foreach statement cannot operate on variables of type 'IEnumerable<IEdmStructuralProperty>' because 'IEnumerable<IEdmStructuralProperty>' does not contain a public definition for 'GetEnumerator'	ConsoleApp1	Reference.tt	1288	
Error		Compiling transformation: foreach statement cannot operate on variables of type 'IEnumerable<IEdmEntitySet>' because 'IEnumerable<IEdmEntitySet>' does not contain a public definition for 'GetEnumerator'	ConsoleApp1	Reference.tt	1490	
Error		Compiling transformation: foreach statement cannot operate on variables of type 'IEnumerable<IEdmSingleton>' because 'IEnumerable<IEdmSingleton>' does not contain a public definition for 'GetEnumerator'	ConsoleApp1	Reference.tt	1530	
Error		Compiling transformation: Operator '==' cannot be applied to operands of type 'method group' and 'string'	ConsoleApp1	Reference.tt	2066	
Error		Compiling transformation: foreach statement cannot operate on variables of type '?' because '?' does not contain a public definition for 'GetEnumerator'	ConsoleApp1	Reference.tt	2077	
Error		Compiling transformation: Argument 1: cannot convert from 'System.Globalization.CultureInfo' to 'string'	ConsoleApp1	Reference.tt	2456	

`

File: ODataConnectedService/src/Templates/ODataT4CodeGenerator.ttinclude [hunk 1]
```diff
@@ -14,8 +14,10 @@ THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 <#@ template debug="true" hostSpecific="true" visibility="internal" linePragmas="false"#>
 <#@ output extension=".cs" #>
 <#@ Assembly Name="System.Core.dll" #>
+<#@ Assembly Name="System.Runtime.dll" #>
 <#@ Assembly Name="System.Xml.dll" #>
 <#@ Assembly Name="System.Xml.Linq.dll" #>
+<#@ Assembly Name="System.Xml.ReaderWriter.dll" #>
```
Comment (AlanWong-MS) [inline ODataConnectedService/src/Templates/ODataT4CodeGenerator.ttinclude:20]: What APIs or classes were missing that require the addition of `System.Runtime.dll` and `System.Xml.ReaderWriter.dll`?

Comment (tchuan) [reply ODataConnectedService/src/Templates/ODataT4CodeGenerator.ttinclude:20]: Alan, please see the comment above.

Comment (AlanWong-MS) [reply ODataConnectedService/src/Templates/ODataT4CodeGenerator.ttinclude:20]: Just saw it as we commented around same time. :)

**Review by AlanWong-MS — APPROVED** [review]

**Review by biaol-odata — APPROVED** [review]

**Pull request merged**